src/tarea.ts

```diff
@@ -0,0 +1,13 @@
+enum Urgencia {
+    Baja,
+    Media,
+    Alta,
+    MuyAlta
+}
+
+interface Tarea {
```

Comment:
Este código sigue teniendo como mensaje de commit "update tarea.ts".

Reply:
Y aquí ya te lo dije.
También es el mensaje por omisión que se pone cuando se edita desde la web. En el objetivo 2 es conveniente que empecéis a manejar con soltura un IDE, y trabajar con git desde la línea de órdenes.